File: Nivel Básico: Introducción a la programación/C++/11451 El perrito que quiere un hueso.cpp
#include <iostream>
using namespace std;

int main() {
  int L1; cin >> L1;
  int T1; cin >> T1;
  int L2; cin >> L2;
  int T2; cin >> T2;
  
  if(L1 > L2 && T1 > T2){
    cout << "Hueso 1";
  }else if(L1 < L2 && T1 < T2){
    cout << "Hueso 2";
  }else{
    cout << "Perrito confundido :(";
  }
}

File: Nivel Básico: Introducción a la programación/Python/11451 El perrito que quiere un hueso.py
def _main() -> None:
  H1 = input().split(' ')
  H2 = input().split(' ')
  L1 = int(H1[0])
  T1 = int(H1[1])
  L2 = int(H2[0])
  T2 = int(H2[1])

  if L1 > L2 and T1 > T2:
    print('Hueso 1')
  elif L1 < L2 and T1 < T2:
    print('Hueso 2')
  else:
    print('Perrito confundido :(')

  pass

if __name__ == '__main__':
  _main()
